README.md

```diff
@@ -262,6 +264,65 @@ func main() {
 }
 ```
 
+### Send a mail through a proxy
+``` go
+package main
+
+import (
+	"github.com/mailjet/mailjet-apiv3-go"
+	"github.com/mailjet/mailjet-apiv3-go/resources"
+
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+)
+
+// Set the http client with the given proxy url
+func setupProxy(url string) *http.Client {
```

Comment:
@HT808s any reason not to include this helper method directly in the wrapper rather than having users to copy it in their own codebase?